dev/lint-requirements.txt

```diff
@@ -1,7 +1,5 @@
-prospector[with_pyroma]==0.12.7
-pep8==1.7.1
 pyarrow==0.17.0
-pylint==1.8.2
+pylint==2.11.1
```

Comment:
Pin `pylint` since it's probably exhausting to fix errors everytime a new version of pylint is released.